eclair-core/src/main/scala/fr/acinq/eclair/payment/send/PaymentInitiator.scala

```diff
@@ -362,6 +362,7 @@ object PaymentInitiator {
                                paymentRequest: Option[PaymentRequest],
                                storeInDb: Boolean, // e.g. for trampoline we don't want to store in the DB when we're relaying payments
                                publishEvent: Boolean,
+                               recordMetrics: Boolean,
```

Comment:
Need to clarify why we may not want to publish metrics (probably same reason as for `publishEvent`? needs a scaladoc).